lib/wilbertils/error_handler.rb

```diff
@@ -0,0 +1,30 @@
+require 'active_support/concern'
+require 'active_support/rescuable'
+
+module Wilbertils
+  module ErrorHandler
+    extend ActiveSupport::Concern
+    include ActiveSupport::Rescuable
+
+    included do
+      rescue_from StandardError, :with => :rescue_error
+    end
+
+    def rescue_error(error, **options)
+      self.class.rescue_error(error, **options)
+    end
+
+    def self.rescue_error(error, **options)
+      log = defined?(logger) ? logger : Rails.logger
```

Comment:
why are these both needed ?

Reply:
So that you can do both `Wilbertils::ErrorHandler.rescue_error(e)` or `include Wilbertils::ErrorHandler` then just `rescue_with_handler(e)`